Lib/test/test_compile.py

```diff
@@ -752,6 +752,12 @@ def continue_in_while():
             self.assertEqual(None, opcodes[0].argval)
             self.assertEqual('RETURN_VALUE', opcodes[1].opname)
 
+    def test_big_dict_literal(self):
+        # See bpo-41531
+        dict_size = 2 * 0xFFFF
```

Comment:
Maybe use 65536 to make the test faster? https://bugs.python.org/issue41531#msg375255